<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PartitionEditorWidgetBase</class>
 <widget class="QWidget" name="PartitionEditorWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeWidget" name="partitionsTreeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QToolButton" name="addButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/wombat.qrc">
           <normaloff>:/icons/24/list-add.png</normaloff>:/icons/24/list-add.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/wombat.qrc">
           <normaloff>:/icons/24/list-remove.png</normaloff>:/icons/24/list-remove.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="refreshButton">
         <property name="text">
          <string>...</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/wombat.qrc">
           <normaloff>:/icons/24/view-refresh.png</normaloff>:/icons/24/view-refresh.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="partsListWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="statusLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetMinimumSize</enum>
       </property>
       <item>
        <widget class="QPushButton" name="verifyButton">
         <property name="text">
          <string>verify</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="createButton">
         <property name="text">
          <string>create</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/wombat.qrc"/>
 </resources>
 <connections/>
</ui>
